docs/getting_started.rst

```diff
@@ -23,6 +23,10 @@ then disconnecting from the server.
 Later in the `Context managers`_ section we will see how to 
 simplify this process through the use of the Python *with* statement.
 
+Note: If you require retrying requests after an HTTP 429 error, the
+``Replay429Adapter`` can be added during the ``Cloudant`` client
+construction and configured with an initial back off and retry count.
```

Comment:
Sounds a little odd to me, maybe " can be added when constructing a Cloudant Client..."